partiql-parser/src/scanner.rs

```diff
@@ -258,9 +423,18 @@ mod test {
                         "Remainder NE"
                     );
                 }
-                (Err(expected_err), Err(actual_err)) => {
-                    // TODO make this less strict with respect to error message
-                    assert_eq!(expected_err, actual_err);
+                (
+                    Err(ParserError::SyntaxError {
+                        position: expected_position,
+                        ..
+                    }),
+                    Err(ParserError::SyntaxError {
+                        position: actual_position,
+                        ..
+                    }),
+                ) => {
+                    // just compare the positions for syntax errors...
+                    assert_eq!(expected_position, actual_position);
```

Comment:
Similar to the other code--this probably suggests we should extract this out now that I think about it.

Reply:
Added #16 to track this.